### DISCUSSION TITLE
Osu! Lazer is not working corrrectly on Windows 11

### DISCUSSION THREAD

**Discussion by Taitranz** [ppy/osu#15187] [Q&A] [answered]
Osu! Lazer has stoppped working on Windows 11 after the most recent Osu! Lazer patch. Task Manager shows that the game is running but no game windows appears. I tried numerous reinstall but it still does not work

**Comment by peppy** [comment, marked as answer]
Going to need logs to do anything.

**Reply by Taitranz** [reply]
How would I go about getting logs

**Reply by peppy** [reply]
*please attach logs here, which are located at:*
- `%AppData%/osu/logs` *(on Windows),*
- `~/.local/share/osu/logs` *(on Linux & macOS).*
- `Android/Data/sh.ppy.osulazer/logs` *(on Android)*,
- on iOS they can be obtained by connecting your device to your desktop and copying the `logs` directory from the app's own document storage using iTunes. (https://support.apple.com/en-us/HT201301#copy-to-computer)

**Reply by Taitranz** [reply]
[database.log](https://github.com/ppy/osu/files/7369951/database.log)
[network.log](https://github.com/ppy/osu/files/7369952/network.log)
[performance.log](https://github.com/ppy/osu/files/7369953/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7369954/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7369956/updater.log)

**Reply by peppy** [reply]
Are you starting the game with some weird shortcut? It looks like it's trying to import some beatmap

```csharp
2021-10-18 23:06:11 [verbose]: [?????] [BeatmapOnlineLookupQueue] Online retrieval failed for -  (Taitranz) (Exception of type 'osu.Game.Online.API.APIException' was thrown.)
```

And potentially falling over on it.

**Reply by Taitranz** [reply]
No, I am using the shortcut that the game installs

**Reply by Taitranz** [reply]
I recorded a video 

https://www.youtube.com/watch?v=jAvIH3t5-Yg

**Reply by peppy** [reply]
Please try renaming the `client.db` file from your osu! folder to something else. This will reset all of your client beatmaps, so keep that in mind.

If that helps, please also attach the database to this thread so we can figure what went wrong.

**Reply by Taitranz** [reply]
I am not sure where the client.db file is located

**Reply by peppy** [reply]
If you could find the logs, it should be close.

**Reply by Taitranz** [reply]
![image](https://user-images.githubusercontent.com/88599441/137846254-a5e8e2ea-8cdf-49e6-a18f-2089a8ef66ac.png)
I'm sorry I am not sure which one is client.db

**Reply by Taitranz** [reply]
Should I be searching in Osu! Lazer folder ?

**Reply by Taitranz** [reply]
![image](https://user-images.githubusercontent.com/88599441/137847127-aa322dab-c623-43b7-aa2b-c0fe11d6af7f.png)

**Reply by Taitranz** [reply]
[runtime.log](https://github.com/ppy/osu/files/7370319/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7370320/updater.log)
[database.log](https://github.com/ppy/osu/files/7370321/database.log)
[network.log](https://github.com/ppy/osu/files/7370322/network.log)
[performance.log](https://github.com/ppy/osu/files/7370323/performance.log)

**Reply by Taitranz** [reply]
these logs are of an different osu folder that does contain client.db

**Reply by peppy** [reply]
They are the same logs. Please follow the instructions of renaming client.db and trying again. It's still trying to import a weird beatmap on startup. I'm still suspicious of your game shortcut. You should try running the game directly from `AppData\Local\osulazer\osu!.exe`

**Reply by Taitranz** [reply]
I recorded it so it can be double checked by you
https://www.youtube.com/watch?v=pTVx7IB62OM

**Reply by Taitranz** [reply]
[network.log](https://github.com/ppy/osu/files/7370408/network.log)
[performance.log](https://github.com/ppy/osu/files/7370409/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7370410/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7370411/updater.log)
[database.log](https://github.com/ppy/osu/files/7370412/database.log)

**Reply by peppy** [reply]
I have no idea what's going on, but your logs are showing that osu! is trying to import a beatmap with your username and no artist/title. Are you using the editor at all? Have you previously used the editor? Do you have any osz files laying around in weird places?

Can you try restarting your PC if you haven't already? There may be more than once copies of lazer stuck, which could trigger an import rather than game start scenario.

**Reply by Taitranz** [reply]
Hi, I have solved the problem by deleting both osu in %appdata% and local.

**Reply by Taitranz** [reply]
Uninstalling Osu! Lazer through control panel does not remove those files
